framework/src/main/java/org/apache/drill/test/framework/CmdParam.java

```diff
@@ -52,10 +52,13 @@
     @Parameter(names = {"-r", "--report"}, description = "Generate json report", required=false)
     public boolean generateReports = false;
 
+    @Parameter(names = {"-if", "--includeFailed"}, description = "Include tests tagged as failures", required=false)
```

Comment:
now that we are only going to run failed tests, may be we should rename this to -rf, --runFailed. Also change description